packages/cli/src/environments.ee/source-control/source-control-scoped.service.ts

```diff
@@ -79,6 +80,21 @@ export class SourceControlScopedService {
 		});
 	}
 
+	async getCredentialsInAdminProjectsFromContext(
```

Comment:
I'm not sure where we use this function?

Reply:
Ah yes, nice catch. i introduced it because I was thinking of handling credentials as well in this PR, but we can do it later